arkane/ess/factoryTest.py

```diff
@@ -37,23 +37,23 @@
 from rmgpy.exceptions import InputError
 
 from arkane.ess import GaussianLog, MolproLog, OrcaLog, QChemLog, TeraChemLog
-from arkane.util import determine_qm_software
+from arkane.ess.factory import ess_factory
 
 ################################################################################
 
 
 class TestThermo(unittest.TestCase):
     """
-    Contains unit tests of the util module.
+    Contains unit tests of the factory module.
     """
     @classmethod
     def setUpClass(cls):
         """
         A method that is run before all unit tests in this class.
         """
-        cls.data_path = os.path.join(os.path.dirname(__file__), 'data')
+        cls.data_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '../data'))
```

Comment:
`../` won't work for Windows, which although we don't support, it is better to be OS agnostic if we can. I think you can achieve the same result using `os.path.dirname` or related

Reply:
Can't we just support the better OS? xD https://www.powermax.com/the-top-ten-reasons-macs-are-better-than-pcs/ 

kidding I updated it